src/virtualenv/activation/powershell/__init__.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import absolute_import, unicode_literals
+
+from virtualenv.util.path import Path
+
+from ..via_template import ViaTemplateActivator
+
+
+class PowerShellActivator(ViaTemplateActivator):
+    def templates(self):
+        yield Path("activate.ps1")
```

Comment:
Powershell works on stuff beyond Windows?

Reply:
yep, also works on linux -- I tried it out for [aoc 2019](https://github.com/anthonywritescode/aoc2019/blob/master/day10/part2.ps1)

Reply:
yes 👍 for the last two years or so

Reply:
TIL